src/Entity/ApiToken.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ApiTokenRepository;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ApiTokenRepository::class)]
+#[ORM\Table(name: "user_api")]
+class ApiToken
+{
+    #[ORM\Id]
+    #[ORM\Column(type: 'string', length: 50)]
+    private $userId;
```

Comment:
Can we set these private fields as strings?

Reply:
My bad :)